### PR TITLE
Added a settings page and set up some facebook stuff

### PR DESCRIPTION
**Pull request by AndrewKishino**

### CODE REVIEW

File: client/app/services/services.js [hunk 1]
```diff
@@ -270,4 +270,55 @@ angular.module('battlescript.services', [])
...
+    return deferred.promise;
+  }
+
+  return {
+    getMyLastName: getMyLastName,
+    getUserID: getUserID,
+    getUserImg: getUserImg
+  }
+
 });
```
Comment (codydaig) [inline client/app/services/services.js]: New Line

File: client/app/settings/settings.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+      </div>
+      </br>
+      <fb:login-button size="large" >Find Friends with Facebook</fb:login-button>
+
+    </div><!-- dashboard -->
+  </div><!-- /container -->
+</main><!-- /main -->
+
+<footer-partial></footer-partial>
```
Comment (codydaig) [inline client/app/settings/settings.html]: New Line

File: client/assets/css/settings.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.settings__title__bold {
+  margin: 0;
+  padding: 0;
+  color: #555;
+  font-weight: 700;
+  line-height: 1.2;
+  font-size: 30px;
+}
```
Comment (codydaig) [inline client/assets/css/settings.css]: New Line

**Comment by codydaig** [comment]
LGTM

**Comment by Kevbyte** [comment]
LGTM

**Pull request merged**